README.md

```diff
@@ -1 +1,48 @@
-# Portfolio_Mobile_Version
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Personal Portfolio Page
+
+> This is the mobile version of a personal portfolio page that shows of my accomplishments as a software developer.
+
+![screenshot](assets/screenshot.PNG)
+
+## Built With
+
+- HTML and CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- A simple web browser (preferably Chrome or Firefox)
+- An IDE
+
+### Setup
+- Download this repository and open index.html file.
+
+## Authors
```

Comment:
- [ ] Kindly add all steps to get the project locally. try to explain how a user can clone the repo, what code he can run to install all dependencies and finally how he can layout the index